probe-rs/src/bin/probe-rs/main.rs

```diff
@@ -110,7 +110,7 @@ pub struct FormatOptions {
     /// If a target has a preferred format, we use that.
     /// Finally, if neither of the above cases are true, we default to ELF.
     #[clap(value_enum, ignore_case = true, long)]
-    #[serde(deserialize_with = "format_from_str")]
+    #[serde(deserialize_with = "format_from_str", alias = "format")]
```

Comment:
I'd like to see a todo to clean this up eventually :)

Reply:
Done :)